SDWebImage/Core/SDImageCoderHelper.h

```diff
@@ -90,6 +90,12 @@
 + (UIImage * _Nullable)decodedAndScaledDownImageWithImage:(UIImage * _Nullable)image limitBytes:(NSUInteger)bytes;
 
 #if SD_UIKIT || SD_WATCH
+/**
+ Control the default limit bytes to scale down larget images.
+ This value must be larger than or equal to 1MB. Defaults to 60MB.
+ */
+@property (class, readwrite) NSUInteger defaultScaleDownLimitBytes;
+
```

Comment:
One strange question:
Why all the force decode method (`decodedImageWithImage` and `decodedAndScaledDownImageWithImage:limitBytes:`) only applied on iOS/tvOS/watchOS ?
Those methods use Core Graphics, and macOS have the same API as well. macOS still use Core Animation for rendering, so I don't think they should be ignored.

Reply:
What about we change those method to let it works on macOS as well.

The default limit bytes on macOS can be doubled, like 120MB (MacBook's RAM is 4GB+)

Reply:
That's a good idea!